<?xml version="1.0" encoding="UTF-8"?><ui version="4.0"><class>GenDialog</class><widget class="QDialog" name="GenDialog"><property name="windowModality"><enum>Qt::ApplicationModal</enum></property><property name="geometry"><rect><x>0</x><y>0</y><width>480</width><height>160</height></rect></property><property name="windowTitle"><string>Сгенерировать исходные данные случайно</string></property><widget class="QDialogButtonBox" name="buttonBox"><property name="geometry"><rect><x>10</x><y>130</y><width>460</width><height>27</height></rect></property><property name="orientation"><enum>Qt::Horizontal</enum></property><property name="standardButtons"><set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set></property><property name="centerButtons"><bool>true</bool></property></widget><widget class="QSpinBox" name="spinPoints"><property name="geometry"><rect><x>170</x><y>10</y><width>300</width><height>28</height></rect></property><property name="sizePolicy"><sizepolicy hsizetype="Minimum" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="correctionMode"><enum>QAbstractSpinBox::CorrectToNearestValue</enum></property><property name="maximum"><number>1000000</number></property><property name="singleStep"><number>10</number></property><property name="value"><number>250</number></property></widget><widget class="QLabel" name="labelKind"><property name="geometry"><rect><x>5</x><y>45</y><width>160</width><height>20</height></rect></property><property name="sizePolicy"><sizepolicy hsizetype="Preferred" vsizetype="Preferred"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy></property><property name="text"><string>Вид фигуры:</string></property><property name="alignment"><set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set></property><property name="wordWrap"><bool>false</bool></property></widget><widget class="QComboBox" name="comboKind"><property name="geometry"><rect><x>170</x><y>40</y><width>300</width><height>28</height></rect></property><property name="sizePolicy"><sizepolicy hsizetype="Minimum" vsizetype="Fixed"><horstretch>0</horstretch><verstretch>0</verstretch></sizepolicy></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><item><property name="text"><string>Эллипс</string></property></item><item><property name="text"><string>Прямоугольник</string></property></item></widget><widget class="QLabel" name="labelPoints"><property name="geometry"><rect><x>5</x><y>15</y><width>160</width><height>20</height></rect></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="text"><string>Количество точек:</string></property><property name="alignment"><set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set></property></widget><widget class="QLabel" name="labelX"><property name="geometry"><rect><x>5</x><y>75</y><width>160</width><height>20</height></rect></property><property name="text"><string>Диапазон по X:</string></property><property name="alignment"><set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set></property></widget><widget class="QLabel" name="labelY"><property name="geometry"><rect><x>5</x><y>105</y><width>160</width><height>20</height></rect></property><property name="text"><string>Диапазон по Y:</string></property><property name="alignment"><set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set></property></widget><widget class="QDoubleSpinBox" name="spinXfrom"><property name="geometry"><rect><x>170</x><y>70</y><width>150</width><height>28</height></rect></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="correctionMode"><enum>QAbstractSpinBox::CorrectToNearestValue</enum></property><property name="decimals"><number>6</number></property><property name="minimum"><double>-1000000000.000000000000000</double></property><property name="maximum"><double>1000000000.000000000000000</double></property><property name="singleStep"><double>0.500000000000000</double></property></widget><widget class="QDoubleSpinBox" name="spinXto"><property name="geometry"><rect><x>320</x><y>70</y><width>150</width><height>28</height></rect></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="correctionMode"><enum>QAbstractSpinBox::CorrectToNearestValue</enum></property><property name="decimals"><number>6</number></property><property name="minimum"><double>-1000000000.000000000000000</double></property><property name="maximum"><double>1000000000.000000000000000</double></property><property name="singleStep"><double>0.500000000000000</double></property><property name="value"><double>5.000000000000000</double></property></widget><widget class="QDoubleSpinBox" name="spinYfrom"><property name="geometry"><rect><x>170</x><y>100</y><width>150</width><height>28</height></rect></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="correctionMode"><enum>QAbstractSpinBox::CorrectToNearestValue</enum></property><property name="decimals"><number>6</number></property><property name="minimum"><double>-1000000000.000000000000000</double></property><property name="maximum"><double>1000000000.000000000000000</double></property><property name="singleStep"><double>0.500000000000000</double></property><property name="value"><double>0.000000000000000</double></property></widget><widget class="QDoubleSpinBox" name="spinYto"><property name="geometry"><rect><x>320</x><y>100</y><width>150</width><height>28</height></rect></property><property name="maximumSize"><size><width>16777215</width><height>16777215</height></size></property><property name="alignment"><set>Qt::AlignCenter</set></property><property name="correctionMode"><enum>QAbstractSpinBox::CorrectToNearestValue</enum></property><property name="decimals"><number>6</number></property><property name="minimum"><double>-1000000000.000000000000000</double></property><property name="maximum"><double>1000000000.000000000000000</double></property><property name="singleStep"><double>0.500000000000000</double></property><property name="value"><double>5.000000000000000</double></property></widget></widget><resources/><connections><connection><sender>buttonBox</sender><signal>accepted()</signal><receiver>GenDialog</receiver><slot>accept()</slot><hints><hint type="sourcelabel"><x>248</x><y>254</y></hint><hint type="destinationlabel"><x>157</x><y>274</y></hint></hints></connection><connection><sender>buttonBox</sender><signal>rejected()</signal><receiver>GenDialog</receiver><slot>reject()</slot><hints><hint type="sourcelabel"><x>316</x><y>260</y></hint><hint type="destinationlabel"><x>286</x><y>274</y></hint></hints></connection></connections></ui>
